generator.yml

```diff
@@ -286,9 +287,8 @@ builds:
   - {<< : *i386,          << : *next,             << : *llvm_full,       boot: true,  llvm_version: *llvm_latest}
   - {<< : *mips,          << : *next,             << : *mips_llvm,       boot: true,  llvm_version: *llvm_latest}
   - {<< : *mipsel,        << : *next,             << : *llvm,            boot: true,  llvm_version: *llvm_latest}
-  # ppc32 and ppc64: Build disabled (https://github.com/ClangBuiltLinux/continuous-integration2/issues/2)
-  # - {<< : *ppc32,         << : *next,             << : *llvm,            boot: true,  llvm_version: *llvm_latest}
-  # - {<< : *ppc64,         << : *next,             << : *ppc64_llvm,      boot: true,  llvm_version: *llvm_latest}
+  - {<< : *ppc32,         << : *next,             << : *llvm,            boot: true,  llvm_version: *llvm_latest}
+  - {<< : *ppc64,         << : *next,             << : *ppc64_llvm,      boot: true,  llvm_version: *llvm_latest}
```

Comment:
it looks the ppc builds are re-enabled (separate commit, plz) for next, but not the other trees. Intentional?

Reply:
No, these are not re-enabled technically, these are being copied from llvm_tot, where they are enabled, The diff is weird.